bundler/lib/bundler/fetcher.rb

```diff
@@ -47,7 +47,7 @@ def initialize(remote_uri)
         remote_uri = filter_uri(remote_uri)
         super "Authentication is required for #{remote_uri}.\n" \
           "Please supply credentials for this source. You can do this by running:\n" \
-          " bundle config set #{remote_uri} username:password"
+          " bundle config set --global #{remote_uri} username:password"
```

Comment:
In the other places, you chose `--local` for configuring credentials for a source. Should we use that here too?

Reply:
Done, but might change again for https://github.com/rubygems/rubygems/pull/3917#discussion_r479780099

Reply:
Yeah, I saw your other comment after I posted this.